### ISSUE TITLE
DeepSVDD get 84% AUROC on SWaT, better than GANF

### ISSUE THREAD

**Issue by zqhang** [EnyanDai/GANF#1]
I'm following your great work. However, when I run the DeepSVDD provided in the code, I get 84% AUROC on SWaT, better than GANF.  It seems that  DeepSVDD overfits the dataset. How can I solve this? The settings are as follows. I wish you can provide the setting of DeepSVDD so that I can continue following your great work. Thank you very much.  
epochs = 40
input_feature = 51
hidden_size = 64
If possible, could you send your trianing code for baseline models to my email  zhouqihang716@gmail.com
![image](https://user-images.githubusercontent.com/95111581/164728230-0d3bc236-1297-4f7d-8cc6-615b4e9db16b.png)

**Comment by knowonething** [comment]
> 
Can you tell me your code running environment? In the forward function of GNN, I have some problems:
torch.einsum('nkld,kj->njld',h,A)
The values in its output are all nan. I don't know if this is due to the wrong version of some python library.

**Comment by EnyanDai** [comment]
The training of DeepSVDD can be tricky. There are several factors can affect such as whether pretraining or not. The number training epochs. And we also observe that the random seed can affect a lot due to its sensitivity to the initialization. I will share the code to you.
